app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb

```diff
@@ -5,11 +5,14 @@
     display_link_if_authorized(_('Run all Ansible roles'), hash_for_play_roles_hostgroup_path(id: hostgroup), :'data-no-turbolink' => true, title: _('Run all Ansible roles on hosts belonging to this host group'))
   end
 
+  assign_jobs = link_to(_("Configure Ansible Job"), "/ansible/hostgroups/#{hostgroup.id}", { class: 'la' })
```

Comment:
Shouldn't we check for permissions here?

Reply:
We probably should, changed.